app.py

```diff
@@ -0,0 +1,162 @@
+from flask import Flask, Blueprint, jsonify, request
+import boto3
+from .config import Config
+from .email_sender import EmailSender
+import json 
+import requests
+from flask_marshmallow import Marshmallow
+from blackfynn import Blackfynn
+from .serializer import ContactRequestSchema
+# from pymongo import MongoClient
+import logging
+
+
+app = Flask(__name__)
+ma = Marshmallow(app)
+email_sender = EmailSender()
+mongo = None
+bf = None
+s3 = boto3.client('s3',
+                  aws_access_key_id=Config.SPARC_PORTAL_AWS_KEY,
+                  aws_secret_access_key=Config.SPARC_PORTAL_AWS_SECRET,
+                  region_name='us-east-1'
+                  )
+
+
+@app.before_first_request
+def connect_to_blackfynn():
+    global bf
+    bf = Blackfynn(
+        api_token=Config.BLACKFYNN_API_TOKEN,
+        api_secret=Config.BLACKFYNN_API_SECRET,
+        env_override=False,
+        host=Config.BLACKFYNN_API_HOST
+    )
+
+# @app.before_first_request
+# def connect_to_mongodb():
+#     global mongo
+#     mongo = MongoClient(Config.MONGODB_URI)
+
+
+api_blueprint = Blueprint('api', __name__, url_prefix='/api')
+
+
+@api_blueprint.route('/health')
+def health():
+    return json.dumps({ "status": "healthy" })
+
+
+@api_blueprint.route("/contact", methods=["POST"])
+def contact():
+    data = json.loads(request.data)
+    contact_request = ContactRequestSchema().load(data)
```

Comment:
It is suspicious that I had to change this line (in sparc-portal) it is `contact_request = ContactRequestSchema().load(data).data`